flink/v1.12/flink/src/main/java/com/netease/arctic/flink/write/AdaptHiveFlinkAppenderFactory.java

```diff
@@ -216,6 +228,18 @@ public PositionDeleteWriter<RowData> newPosDeleteWriter(EncryptedOutputFile outp
               .transformPaths(path -> StringData.fromString(path.toString()))
               .buildPositionWriter();
 
+        case ORC:
+          return ORC.writeDeletes(outputFile.encryptingOutputFile())
+              .createWriterFunc((schema, typDesc) -> FlinkOrcWriter.buildWriter(lazyPosDeleteFlinkSchema(), schema))
```

Comment:
> Maybe 'RowType flinkPosDeleteSchema = FlinkSchemaUtil.convert(DeleteSchemaUtil.posDeleteSchema(posDeleteRowSchema))' instead of lazyPosDeleteFlinkSchema()

done